src/elements/kw-social-comment/kw-social-comment.html

```diff
@@ -192,7 +192,7 @@ <h3 shown$="[[!comments.length]]">Be the first to comment!</h3>
                 </div>
                 <div class="content">
                     <p class="comment-header"><span class="author">{{comment.author.username}}</span>
-                      <span class="date">[[_timeSince(comment.date_created)]] ago</span>
+                      <span class="date">[[_timeSince(comment.date_created, comments.*)]] ago</span>
```

Comment:
Maybe we could use `this.notifyPath` here instead of passing a "ghost" second parameter?